### PR TITLE
Added more haskell learning resources

### PR DESCRIPTION
**Pull request by HOOGST**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -433,6 +433,12 @@ When I was in college, I missed a lot of opportunities like hackathons, conferen
 
 ## 1.13 Haskell
  - [Happy Learn Haskell Tutorial](http://www.happylearnhaskelltutorial.com/)
+ - [Functional Programming in Haskell](https://github.com/caiorss/Functional-Programming/blob/master/haskell/README.org)
```
Comment (Contrevien) [inline README.md]: Approved

File: README.md [hunk 2]
```diff
@@ -433,6 +433,12 @@ When I was in college, I missed a lot of opportunities like hackathons, conferen
...
+ - [Adventure with Types in Haskell - Simon Peyton Jones (Lecture 1)](https://youtu.be/6COvD8oynmI?list=RD6COvD8oynmI)
```
Comment (Contrevien) [inline README.md]: Approved

File: README.md [hunk 3]
```diff
@@ -433,6 +433,12 @@ When I was in college, I missed a lot of opportunities like hackathons, conferen
...
+ - [Category Theory for Programmers](https://bartoszmilewski.com/2014/10/28/category-theory-for-programmers-the-preface/)
```
Comment (Contrevien) [inline README.md]: No

File: README.md [hunk 4]
```diff
@@ -433,6 +433,12 @@ When I was in college, I missed a lot of opportunities like hackathons, conferen
...
+ - [Haskell Book](haskellbook.com)
```
Comment (Contrevien) [inline README.md]: Add the $ sign to indicate that it is a paid resource

File: README.md [hunk 5]
```diff
@@ -433,6 +433,12 @@ When I was in college, I missed a lot of opportunities like hackathons, conferen
...
+ - [FUNCTIONAL PROGRAMMING - University of Edinburgh](https://www.inf.ed.ac.uk/teaching/courses/inf1/fp/)
+ - [Haskell Programming from First Principles(Follow-up resources)](https://github.com/pushcx/hpffp-resources)
```
Comment (Contrevien) [inline README.md]: Approved

File: README.md [hunk 6]
```diff
@@ -433,6 +433,12 @@ When I was in college, I missed a lot of opportunities like hackathons, conferen
 
 ## 1.13 Haskell
  - [Happy Learn Haskell Tutorial](http://www.happylearnhaskelltutorial.com/)
+ - [Functional Programming in Haskell](https://github.com/caiorss/Functional-Programming/blob/master/haskell/README.org)
+ - [Adventure with Types in Haskell - Simon Peyton Jones (Lecture 1)](https://youtu.be/6COvD8oynmI?list=RD6COvD8oynmI)
+ - [Category Theory for Programmers](https://bartoszmilewski.com/2014/10/28/category-theory-for-programmers-the-preface/)
+ - [Haskell Book](haskellbook.com)
+ - [FUNCTIONAL PROGRAMMING - University of Edinburgh](https://www.inf.ed.ac.uk/teaching/courses/inf1/fp/)
```
Comment (Contrevien) [inline README.md]: No

File: README.md (hunk 3 above)
Comment (HOOGST) [reply README.md]: Why no?

**Pull request closed without merging**